06-setup-backup-restore/trilio.md

```diff
@@ -152,117 +182,105 @@ Please follow the steps below, to install `TrilioVault` via `Helm`:
 
-    The output looks similar to the following:
+   ```text
+   NAME                                            CHART VERSION   APP VERSION     DESCRIPTION
+   triliovault-operator/k8s-triliovault-operator   2.6.7           2.6.7           K8s-TrilioVault-Operator is an operator designe...
+   ```
+   **Note:**
 
-    ```text
-    NAME                                    CHART VERSION   APP VERSION     DESCRIPTION                                       
-    triliovault/k8s-triliovault             2.6.3           2.6.3           K8s-TrilioVault provides data protection and re...
-    triliovault/k8s-triliovault-operator    0.9.0           0.9.0           K8s-TrilioVault-Operator is an operator designe...
-    ```
+   The chart of interest is `triliovault-operator/k8s-triliovault-operator`, which will install `TrilioVault for Kubernetes` on the cluster along with the `TrilioVault-Manager`. You can run `helm show values triliovault-operator/k8s-triliovault-operator`, and export to a file to see all the available options.
 
-    **Note:**
-
-    The chart of interest is `triliovault/k8s-triliovault`, which will install `TrilioVault for Kubernetes` on the cluster. You can run `helm show values triliovault/k8s-triliovault --version 2.6.3`, and export to a file to see all the available options.
 3. Then, open and inspect the TrilioVault `Helm` values file file provided in the `Starter kit` repository, using an editor of your choice (preferably with `YAML` lint support). You can use [VS Code](https://code.visualstudio.com) for example:
 
-    ```shell
-    TRILIOVAULT_CHART_VERSION="2.6.3"
-
-    code 06-setup-backup-restore/assets/manifests/triliovault-values-v${TRILIOVAULT_CHART_VERSION}.yaml
-    ```
+   ```shell
+   code 06-setup-backup-restore/assets/manifests/triliovault-values.yaml
+   ```
 
 4. Finally, install `TrilioVault for Kubernetes` using `Helm`:
 
-    ```shell
-    TRILIOVAULT_CHART_VERSION="2.6.3"
-
-    helm install triliovault triliovault/k8s-triliovault --version "${TRILIOVAULT_CHART_VERSION}" \
-      --namespace tvk \
-      --create-namespace \
-      -f 06-setup-backup-restore/assets/manifests/triliovault-values-v${TRILIOVAULT_CHART_VERSION}.yaml
-    ```
-
-    **Note:**
-
-    A `specific` version for the `TrilioVault` Helm chart is used. In this case `2.6.3` is picked, which maps to the `2.6.3` version of the application (see the output from `Step 2.`). It’s good practice in general, to lock on a specific version. This helps to have predictable results, and allows versioning control via `Git`.
+   ```shell
+   helm install triliovault-operator triliovault-operator/k8s-triliovault-operator \
+     --namespace tvk \
+     --create-namespace \
+     -f 06-setup-backup-restore/assets/manifests/triliovault-values.yaml
+   ```
+   **Note:**
+   Above command install both `TrilioVault Operator` and `TriloVault Manager` (TVM) Custom Resource using the parameters provided in the `triliovault-values.yaml`.
```

Comment:
Because you removed the `version field` from the `values file name` and from the `helm install` command, I would add some explanations in this note, and let the user know that the application version is now managed by the `tag` field inside the values file.